Handler/Bid.hs

```diff
@@ -92,6 +106,7 @@ bidForm userId mbid = renderSematnicUiDivs $ Bid
         bidders = do
             entities <- runDB $ selectList [] [Asc UserIdent]
             optionsPairs $ map (\item -> (userIdent $ entityVal item, entityKey item)) entities
+        priceField = check validateBidPrice intField
```

Comment:
@bitamar using [check](https://www.stackage.org/haddock/lts-6.16/yesod-form-1.4.7.1/Yesod-Form-Functions.html#v:check) instead of checkBool, seems to be the way forward.

And we can re-use those functions [above](https://github.com/Gizra/circuit-yesod/pull/12/files#diff-365b570ca41e75c12ca40534adc5ff47R67) as-well